packages/kmod/project.bri

```diff
@@ -24,7 +24,19 @@ export default function kmod(): std.Recipe<std.Directory> {
   `
     .workDir(source)
     .dependencies(std.toolchain, meson, ninja, scdoc, openssl)
-    .toDirectory();
+    .toDirectory()
+    .pipe(
+      std.pkgConfigMakePathsRelative,
+      (recipe) =>
+        std.setEnv(recipe, {
+          CPATH: { append: [{ path: "include" }] },
+          LIBRARY_PATH: { append: [{ path: "lib" }] },
+          PKG_CONFIG_PATH: {
+            append: [{ path: "lib/pkgconfig" }, { path: "share/pkgconfig" }],
```

Comment:
This recipe has two pkgconfig folders which contain different files, so I have added both of them.